draft-ietf-ppm-dap.md

```diff
@@ -1638,6 +1638,10 @@ Whether this phase is reached depends on the VDAF: for 1-round VDAFs, like
 Prio3, processing has already completed. Continuation is required for VDAFs
 that require more than one round.
 
+If for whatever reason the Leader must abandon the aggregation job, it SHOULD
+send an HTTP DELETE request to the aggregation job URI so that the Helper knows
+it can clean up its state.
```

Comment:
Since this recommendation applies to the leader, it should be in the "Leader Continuation" section, below.

Reply:
Done!

Reply:
Now I'm being difficult: I think it makes more sense to put this paragraph at the very end of "Leader Continuation" rather than the very beginning.

Reply:
I don't fully agree given that it's a branch point that goes more naturally before constructing the next request. However I don't think it makes much of a practical difference, so I'm happy to follow you here.

Reply:
Moved!